jest.setup.after-env.js

```diff
@@ -1,2 +1,5 @@
 import '@testing-library/jest-dom';
 import 'intersection-observer';
+import React from 'react';
+
+global.React = React;
```

Comment:
With React 18 we actually don't need to import React within each file.

https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports

This is needed to get the tests to pass though. Going to follow up with a little tidy up of the current imports using the script from the link above.